kubernetes/schema_horizontal_pod_autoscaler.go

```diff
@@ -7,6 +7,93 @@ import (
 	autoscalingv2beta2 "k8s.io/api/autoscaling/v2beta2"
 )
 
+func horizontalPodAutoscalerSpecV2() *schema.Schema {
+	return &schema.Schema{
+		Type:        schema.TypeList,
+		Description: "Behaviour of the autoscaler. More info: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#spec-and-status",
+		Required:    true,
+		MaxItems:    1,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"max_replicas": {
+					Type:        schema.TypeInt,
+					Description: "Upper limit for the number of pods that can be set by the autoscaler.",
+					Required:    true,
+				},
+				"metric": {
+					Type:        schema.TypeList,
+					Computed:    true,
+					Optional:    true,
+					Description: "The specifications for which to use to calculate the desired replica count (the maximum replica count across all metrics will be used). The desired replica count is calculated multiplying the ratio between the target value and the current value by the current number of pods. Ergo, metrics used must decrease as the pod count is increased, and vice-versa. See the individual metric source types for more information about how each type of metric must respond. If not set, the default metric will be set to 80% average CPU utilization.",
+					Elem:        metricSpecFields(),
+				},
+				"min_replicas": {
+					Type:        schema.TypeInt,
+					Description: "Lower limit for the number of pods that can be set by the autoscaler, defaults to `1`.",
+					Optional:    true,
+					Default:     1,
+				},
+				"behavior": {
+					Type:        schema.TypeList,
+					Description: "Behavior configures the scaling behavior of the target in both Up and Down directions (`scale_up` and `scale_down` fields respectively).",
+					Optional:    true,
+					Computed:    true,
+					MaxItems:    1,
+					Elem: &schema.Resource{
+						Schema: map[string]*schema.Schema{
+							"scale_up": {
+								Type:        schema.TypeList,
+								Description: "Scaling policy for scaling Up",
+								Optional:    true,
+								Computed:    true,
+								Elem:        scalingRulesSpecFields(),
+							},
+							"scale_down": {
+								Type:        schema.TypeList,
+								Description: "Scaling policy for scaling Down",
+								Optional:    true,
+								Computed:    true,
```

Comment:
Made `spec. behavior.scale_down` computed. But since I also moved the whole `spec` into a diff file\function it is not clear what has changed.